### ISSUE TITLE
[Suggestion] Tasks app

### ISSUE THREAD

**Issue by nift4** [HelloVolla/volla-os-beta-test#13]
I suggest OpenTasks ([F-Droid](https://f-droid.org/packages/org.dmfs.tasks/), [Aurora](https://play.google.com/store/apps/details?id=org.dmfs.tasks), [GitHub](https://github.com/dmfs/opentasks)) because it is licensed under the libre Apache 2 Licence, is supported best by DAVx5 and has an API for other UIs like the Volla Launcher.

**Comment by wurzer** [comment]
I agree. It's a good app. Why do you think should the app be pre-installed?

**Comment by nift4** [comment]
For full Sync experience
